config/types/filesystems.go

```diff
@@ -21,9 +21,9 @@ import (
 )
 
 type Filesystem struct {
-	Name  string `yaml:"name"`
-	Mount *Mount `yaml:"mount"`
-	Path  string `yaml:"path"`
+	Name  string  `yaml:"name"`
```

Comment:
Did this make its way into the wrong commit?

Reply:
Oh these were fixes for issues I found when writing the tests. I'll move them into the correct commit.